android-generator/src/main/scala/models/generator/android/AndroidJavaUtil.scala

```diff
@@ -95,7 +95,7 @@ trait AndroidJavaUtil {
     "double" -> ClassName.get("java.lang","Double"),
     "integer" -> ClassName.get("java.lang", "Integer"),
     "long" -> ClassName.get("java.lang", "Long"),
-    "object" -> ClassName.get("java.util","Map"),
+    "object" -> ClassName.get("java.lang","Object"),
```

Comment:
This doesn't seem right. A json object should be represented as Map[String, Object] in java.

Reply:
reverted for Android/Java